src/fragments/lib/datastore/ios/data-access/save-snippet.mdx

```diff
@@ -1,19 +1,20 @@
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
-Amplify.DataStore.save(
-    Post(title: "My first post",
-         description: "Amplify.DataStore is awesome!",
-         status: .draft)
-) {
-    switch $0 {
-    case .success:
-        print("Created a new post successfully")
-    case .failure(let error):
-        print("Error creating post - \(error.localizedDescription)")
-    }
+do {
+    try await Amplify.DataStore.save(
+        Post(
+            title: "My first post", 
+            content: "Amplify.DataStore is awesome!", 
+            createdAt: .now(),
```

Comment:
looks like the previous code didn't have to pass in createdAt, does the codegenerated Post's initializer for createdAt have a default value? i wonder what's changed

Reply:
the Post model now requires `createdAt` and codegenerated Post's initializer for `createdAt` doesn't have a default value. 
<img width="720" alt="Screen Shot 2022-09-30 at 8 51 16 AM" src="https://user-images.githubusercontent.com/107574718/193308944-ecb69a3d-e467-4651-824e-00b01da08d10.png">